python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/aaa-server-groups.j2

```diff
@@ -13,10 +13,12 @@
 | Server Group Name | Type  | VRF | IP address |
 | ------------------| ----- | --- | ---------- |
 {%     for aaa_server_group in aaa_server_groups %}
-{%         for server in aaa_server_group.servers %}
-{%             set vrf = server.vrf | arista.avd.default('default') %}
+{%         if aaa_server_group.type is arista.avd.defined %}
+{%             for server in aaa_server_group.servers %}
```

Comment:
The servers key is not a required key but it has not been checked "if defined or not in the template", so it throws error if not defined.

Reply:
Fixed.

Reply:
I am not sure this is fixed here - @MaheshGSLAB mentioned that `aaa_server_group.servers` could be unset and I feel like this templates would fail - this probably mean we don't have a good test covering this - maybe lets add one.

Reply:
Fixed.

Reply:
Lets add one tests without servers -

```
- name: RADIUS3
    type: radius
```

Reply:
Added.